lib/components/art/ArtDetailWidget.dart

```diff
@@ -1,5 +1,8 @@
+import 'package:artsideout_app/theme.dart';
 import 'package:flutter/material.dart';
 import 'package:artsideout_app/graphql/Installation.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
```

Comment:
You do not need a new library for one icon that already exists in the built in material library.

https://api.flutter.dev/flutter/material/Icons-class.html
https://material.io/resources/icons/?style=baseline

bookmark is the icon you want from here